hack/upgrade-test-harness/testdata/upcoming/stack.yaml

```diff
@@ -4,7 +4,7 @@ kind: Elasticsearch
 metadata:
   name: es
 spec:
-  version: 7.17.0-SNAPSHOT
+  version: 8.0.0-SNAPSHOT
```

Comment:
It looks like this hasn't been updated before, shouldn't it be `8.2.0-SNAPSHOT`?

Reply:
I think you're right @thbkrkr .  I've updated all of the versions in this file.